Userland/Libraries/LibGUI/SpinBox.cpp

```diff
@@ -63,6 +69,16 @@ void SpinBox::set_value(int value, AllowCallback allow_callback)
     if (m_value == value)
         return;
     m_value = value;
+
+    if (m_value == m_max)
+        m_increment_button->set_enabled(false);
+    else if (m_value == m_min)
+        m_decrement_button->set_enabled(false);
+    else {
+        m_increment_button->set_enabled(true);
+        m_decrement_button->set_enabled(true);
+    }
```

Comment:
If m_value is m_max we don't necessarily enable the decrement button, even though we should. Maybe this could be more like:

```suggestion
    m_increment_button->set_enabled(m_value < m_max);
    m_decrement_button->set_enabled(m_value > m_min);
```

Reply:
Good catch. I was trying to avoid always setting the enabled flag when we didn't need to (the vast majority of cases will be somewhere between min and max I would think. Perhaps that's a silly premature optimization :) I'll get this changed.

Reply:
That's understandable. A lot of methods like `set_enabled()` do an early-out if the new value isn't different from the current one, so you don't need to worry about it here though. :^) https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibGUI/Widget.cpp#L882

Reply:
Makes sense, thanks for the extra info. I've made that change and pushed it :)

Reply:
Never quite knew if I was supposed to say it's resolved or the reviewer :) seems strange that it wouldn't need to be the reviewer confirming the change was made. 

Thanks for the heads up, I've clicked the button :)

Reply:
You click the button then it lets the reviewer know you've resolved their issue. They can request changes again after if needed.